pip/requirements.txt

```diff
@@ -124,14 +124,14 @@ joblib==1.0.1
 jsonpickle==2.0.0
 jsonschema==3.2.0
 jupyter-client==7.0.2
-jupyter-console==6.4.0
+jupyter-console==6.4.3
 jupyter-core==4.7.1
 jupyter-packaging==0.10.4
 jupyterlab-pygments==0.1.2
 jupyter-server==1.10.2
 jupyter-server-mathjax==0.2.3
-jupyterlab-widgets==1.0.1
-keras==2.6.0
+jupyterlab-widgets==1.1.0
+keras==2.8.0
```

Comment:
@aandvalenzuela , looks like this should also match the version of tensorflow. So for now keep it at `2.6.0`